akka-docs/rst/java/logging.rst

```diff
@@ -305,7 +305,7 @@ the first case and ``LoggerFactory.getLogger(String s)`` in the second).
 
   final LoggingAdapter log = Logging.getLogger(system.eventStream(), "my.string");
 
-Logging Thread and Akka Source in MDC
+Logging Thread, Akka Source and Actor System in MDC
```

Comment:
Maybe I'm a bit OCD about those things but aligning the `---` with the title is usually a nice thing to do :)